src/main/lpfConverter.ts

```diff
@@ -6,15 +6,16 @@
 // ==LICENSE-END==
 
 import * as debug_ from "debug";
-import { copy, ensureDir, move } from "fs-extra";
-import { contentType } from "mime-types";
+import { promises as fsp } from "fs";
 import * as moment from "moment";
 import * as os from "os";
-import { basename, extname, join } from "path";
+import { basename, dirname, extname, join } from "path";
+import { Contributor } from "r2-shared-js/dist/es6-es2015/src/models/metadata-contributor";
```

Comment:
TODO: fix `r2-shared-js/dist/es6-es2015/` import (elsewhere as well, not just here)

Reply:
ah damn
automatic import error

can you improve that ?

Reply:
Import path aliases are defined for both TypeScript and WebPack (see code references below).

Perhaps this is a bug in  Visual Studio Code?
https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping

PS: "TypeScript Import Sorter" is the Visual Studio Code extension I personally use, but it does not seem to have an "autofixer" for incorrect import aliasing:
https://github.com/SoominHan/import-sorter

https://github.com/readium/readium-desktop/blob/6a6fad8faa33e5a1c517beda4057ad0f3e5f5314/tsconfig.json#L78-L95

https://github.com/readium/readium-desktop/blob/6a6fad8faa33e5a1c517beda4057ad0f3e5f5314/webpack.config.main.js#L16-L21

https://github.com/readium/readium-desktop/blob/6a6fad8faa33e5a1c517beda4057ad0f3e5f5314/webpack.config.renderer-library.js#L12-L17

https://github.com/readium/readium-desktop/blob/6a6fad8faa33e5a1c517beda4057ad0f3e5f5314/webpack.config.renderer-reader.js#L12-L17